### PR TITLE
TSK-1436: change deleting spaces to removing, add action to move all non-valid requests to correct spaces

### PR DESCRIPTION
**Pull request by ThetaDR**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [x] - Tested for Chrome.
* [ ] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: models/hr/src/index.ts [hunk 1]
```diff
@@ -328,9 +328,9 @@ export function createModel (builder: Builder): void {
   createAction(
     builder,
     {
-      action: view.actionImpl.Delete,
-      label: view.string.Delete,
-      icon: view.icon.Delete,
+      action: view.actionImpl.Archive,
```
Comment (haiodo) [inline models/hr/src/index.ts:331]: Why not add action to core.class.Space on one place?

Comment (ThetaDR) [reply models/hr/src/index.ts:331]: It is there, but I had to override so users couldn't archive head department or department with members (what we had previously set)

Comment (haiodo) [reply models/hr/src/index.ts:331]: Archive is reversible operation, so why not? The question is what to do with Employees being part of archived department.

Comment (ThetaDR) [reply models/hr/src/index.ts:331]: @haiodo unset their department?

**Review by haiodo — APPROVED** [review]

**Pull request merged**